### PR TITLE
AST-1285 - E2E test case for footer builder background color

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added E2E test case for footer builder background color

### Screenshots

https://share.getcloudapp.com/P8u60d68

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/footer-builder/background-color.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/footer-builder/background-color.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/customize';
+import { setBrowserViewport } from '../../../utils/set-browser-viewport';
+import { scrollToElement } from '../../../utils/scroll-to-element';
+describe( 'footer builder backgeround color setting in customizer', () => {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/background-color.test.js:5]: First letter capital

File: tests/e2e/specs/customizer/footer-builder/background-color.test.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+			selector: '.site-footer',
+			property: 'background-color',
+		} ).cssValueToBe( `${ footerBuilder[ 'footer-bg-obj-responsive' ].tablet[ 'background-color' ] }` );
+	} );
+	it( 'background color for mobile should apply correctly', async () => {
+		const footerBuilder = {
+			'section-footer-builder-layout-padding': 60,
+			'footer-bg-obj-responsive': {
+				mobile: {
+					'background-color': 'rgb(210, 248, 249)',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/background-color.test.js:67]: Can we check different colors for responsive devices?

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/background-color.test.js:67]: Okay

File: tests/e2e/specs/customizer/footer-builder/background-color.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/background-color.test.js]: Done with changes

File: tests/e2e/specs/customizer/footer-builder/background-color.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/background-color.test.js]: Added different colors

**Review by premanshup — APPROVED** [review]

**Pull request merged**